src/components/Tokens/TokenDetails/MobileBalanceSummaryFooter.tsx

```diff
@@ -86,6 +87,7 @@ export default function MobileBalanceSummaryFooter({ token }: { token: Currency
   const formattedBalance = formatCurrencyAmount(balance, NumberType.TokenNonTx)
   const formattedUsdValue = formatCurrencyAmount(useStablecoinValue(balance), NumberType.FiatTokenStats)
   const chain = CHAIN_ID_TO_BACKEND_NAME[token.chainId].toLowerCase()
+  const { value: enabled } = useGate('web_dummy_gate_amplitude_id')
```

Comment:
nit:
instead of `enabled` I prefer more explicitly naming like `isDummyGateFlagEnabled`

Reply:
suggestion taken